hello.py

```diff
@@ -1,4 +1,12 @@
 #!/usr/bin/env python
 
-print("Hello, World!")
+for i in range(0,10):
+    if (i % 3):
```

Comment:
We created a public channel for you at [gitter.im/mfellner/hello-world-pr6-fizz-buzz](gitter.im/mfellner/hello-world-pr6-fizz-buzz)
<img src="https://camo.githubusercontent.com/e2df1424c5910135e3bea59b45e22c5ce4abc5b0/68747470733a2f2f6261646765732e6769747465722e696d2f7a616c616e646f2f7a617070722d6d6f636b2e737667" alt="Join the chat at https://gitter.im/zalando/zappr-mock" data-canonical-src="https://badges.gitter.im/zalando/zappr-mock.svg" style="max-width:100%;">